daemons/gptp/common/ptp_message.cpp

```diff
@@ -1364,18 +1366,19 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	 * than a certain number of lost FUPs we should set asCapable
 	 * to false
 	 */
+	diffLastSeqId = sequenceId - port->getLastSeqId();
```

Comment:
It would be nice to have a test framework to inject some of these wrap cases into the gPTP source code!